lib/media-helper.js

```diff
@@ -84,9 +84,6 @@ export function writeScreenshot( data, prefix, i18nScreenshot ) {
 	}
 
 	let directoryName = 'screenshots';
-	if ( i18nScreenshot === true ) {
```

Comment:
Ah, right right.  That makes sense.  Not necessary, but I think it's probably worth putting the envvar here too just for consistency.  And in case we ever run the i18n tests locally with just mocha and not magellan it'll make things less confusing.